docs/guide/shop_features/product_management.md

```diff
@@ -0,0 +1,24 @@
+# Product management
+
+## Product
+
+[[= product_name =]] offers a built-in [Product Content Type](../catalog/product.md).
+
+Any content of this type can be automatically used for all shop functionalities,
+such as adding to basket, price calculation and ordering,
+and can be embedded on the front end with quick ordering buttons.
+
+## Product variants
+
+A product can have different variants, corresponding for example to different colors or sizes.
+
+You can select and preview variants on the product's page.
+Variants can have one or two levels.
+When you choose a first attribute, the shop narrows down the options for the second level.
```

Comment:
```suggestion
When you choose the first attribute, the shop narrows down the options for the second level.
```
Could it be:
"When you choose an option for the attribute at the first level, the shop narrows down the options for the second level."
?